decentraland/unity-renderer
# style: interactions hints and profile button feedback

This PR includes the new style for the interaction hints messages and the highlight feedback for the main menu button.

<img width="1593" alt="Screen Shot 2021-12-29 at 11 08 48" src="https://user-images.githubusercontent.com/51088292/147673438-bae0c443-416a-4295-b4c9-7b4b6bc4abe9.png">

**How to test?**
Test link: https://play.decentraland.zone/?renderer-branch=style/InteractionsHints
1- Place the mouse over the menu button (in the right top corner) and check that a stroke gradient is displayed.
2- Aim an interactive thing in the world (an avatar, events columns in the Genesis Plaza, NPC) and check that the new style is applied (transparent black background with white icons and text near the cursor).
